docs/src/main/asciidoc/application-configuration-guide.adoc

```diff
@@ -148,14 +148,14 @@ public class GreetingResourceTest {
 
 == Package and run the application
 
-Run the application with: `mvn compile quarkus:dev`.
+Run the application with: `./mvnw compile quarkus:dev`.
```

Comment:
That means that we need to add the wrapper to all our quickstarts, no? (and in each project, not at the root)

Reply:
Yes, I will follow up with a PR in the quickstarts

Reply:
Quickstart PR is here: https://github.com/quarkusio/quarkus-quickstarts/pull/104